test/functional/fixtures/tty-test.c

```diff
@@ -6,6 +6,9 @@
 #include <stdlib.h>
 #include <uv.h>
 
+// -V:STRUCT_CAST:641
+#define STRUCT_CAST(Type, obj) ((Type *)(obj))
```

Comment:
This does not hide anything, just silences some warnings. If I am not mistaking regarding what strict aliasing means then you may search for STRUCT_CAST to find code which for sure violates strict aliasing rules. It is libuv which makes us play with struct pointers though.